kitchen.dokken.yml

```diff
@@ -45,7 +45,7 @@ platforms:
 
   - name: centos-8
     driver:
-      image: dokken/centos-8
+      image: dokken/centos-stream-8
```

Comment:
I'd rather this stays the same and just add a new os platform specifically testing this for now.

Reply:
We'll have to skip testing with CentOS 8 then due to the systemd/kernel mismatch which causes inspec to fail to detect service state. I'll switch it for normal 8 in the CI config for the time being and add a new platform.

Reply:
Actually, I don't think it checks service state so we should be good.